packages/app/src/cli/models/app/app.ts

```diff
@@ -331,6 +332,17 @@ export class App implements AppInterface {
     if (complianceWebhooksConfigIndex > -1) {
       setPathValue(this.realExtensions, `${complianceWebhooksConfigIndex}.configuration.webhooks.subscriptions`, [])
     }
+
```

Comment:
is this method `filterDeclarativeWebhooksConfig` actually necessary?
The specification is filtered in the backend, if the CLI doesn't receive the spec it won't load any subscription. So there is no need to filter them here.

But do we need to remove them from the `webhooks` and `privacy-compliance-webhooks` too?

Reply:
I think removing this will cause `webhook_subscriptions` registrations to be created in the database even if the beta is off - I did a 🎩 to confirm and saw the `webhook_subscriptions` registration in the db & in the network call after I `deploy` with the beta changes removed. I think it doesn't show up on the Versions page because there is also an additional beta check on the frontend

on Versions Page:
<img width="954" alt="Screenshot 2024-05-09 at 2 10 32 PM" src="https://github.com/Shopify/cli/assets/32424276/61177b2c-2296-4f43-9d89-38cc4be08104">

Network call, the `handle` with the hashstring is the webhook subscription module
<img width="1039" alt="Screenshot 2024-05-09 at 2 11 26 PM" src="https://github.com/Shopify/cli/assets/32424276/fc6cfe42-2959-46a0-b45b-ec1aaa2dce94">

Database 
<img width="855" alt="Screenshot 2024-05-09 at 2 12 50 PM" src="https://github.com/Shopify/cli/assets/32424276/385e521a-a773-4874-aeef-08b04140059f">